tests/testsuite/cargo_remove/offline/mod.rs

```diff
@@ -0,0 +1,35 @@
+use cargo_test_support::compare::assert_ui;
+use cargo_test_support::curr_dir;
+use cargo_test_support::CargoCommand;
+use cargo_test_support::Project;
+use cargo_test_support::TestEnv;
+
+use crate::cargo_remove::init_registry;
+
+#[cargo_test]
+fn case() {
+    init_registry();
+    let project = Project::from_template(curr_dir!().join("in"));
+    let project_root = project.root();
+    let cwd = &project_root;
+
+    // run the metadata command to populate the cache
+    let cargo = std::env::var_os("CARGO").unwrap();
+    snapbox::cmd::Command::new(cargo)
+        .test_env()
```

Comment:
I think this is using the wrong cargo. This is using the cargo that launched the testsuite, not the one that is being tested.

I'm not sure if there was a specific reason to avoid the `cargo_ui` method, but I think something like this should probably work:

```suggestion
    snapbox::cmd::Command::cargo_ui()
```

Reply:
Will open a fix for this tomorrow.